src/Containers/Lobby/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Col, Row } from 'antd';
+
+import { Box, CenteredContainer, CenteredDiv, Tags } from 'Components/CommonComponents';
+import Button from 'Components/Button';
+import { PlayerList, StartButton } from './styles';
+import { useState } from 'react';
+
+const Lobby = () => {
+  const gameCode = localStorage.getItem('game_code') as string;
+
+  const [playerCount, setPlayerCount] = useState(1);
+
+  return (
+    <Box>
+      <CenteredContainer>
+        <Row gutter={[0, 15]}>
+          <Col span={16} offset={4}>
+            <Row gutter={[10, 20]}>
+              <Col span={12}>
+                <Button
+                  type={'ghost'}
+                  onClick={() => navigator.clipboard.writeText(gameCode)}
+                  style={{ fontFamily: 'Arial' }}
```

Comment:
Why do we use `ghost` button type here?

Reply:
Oh, sorry i forgot about this, when I was looking for a solution to the error with the button 🤓